src/containers/VisitPage.js

```diff
@@ -107,50 +108,63 @@ class VisitPage extends Component {
     this.setState({ client: nextClient });
   };
 
-  resetClient = () => {
+  resetClient = (shouldReset) => {
+
+    if (!shouldReset) {
+      return;
+    }
```

Comment:
Calling `resetClient()` when you don't actually want to reset the client is a little weird. Not a big deal though.

Reply:
Yeah, I know. I was hoping to save on duplication or on having to make a `resetIfOk` method or some such. Never came to a solid conclusion on that.

Reply:
Maybe I'll just change the name of that function itself, lol